### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Header_ScreenReader:Narrator announcing search box as table.

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/vscode-docs#5100]
**Actual**:
Narrator announcing search box as table.

**Expected Result**:
Narrator should announce search box as edit search box. 

**GitHubTags**:

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Narrator;#WCAG1.3.1; #FTP; #STP; #NVDA; #ChromiumEdge;

**Environment Details**:
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps**:

1. Launch the application Visual Studio Code Service.
2. Tab navigate to top navigation bar.
3. Observe that narrator announcing search box as table.

**User Impact**:
Screen reader user will face difficulty while accessing control as narrator reading search box as table.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Narrator Attachment**:
![A11y_Visual Studio Code Services_Home_Header_ScreenReader](https://user-images.githubusercontent.com/97668775/149881241-cb2e12ca-df97-4ab7-a567-02b155e54e8a.gif)

**NVDA ScreenReader Attachment**:
![A11y_Visual Studio Code Services_Home_Header_ScreenReader](https://user-images.githubusercontent.com/97668775/150073193-fa239e5a-da9c-4e15-9292-3aa0c38dab1d.png)

**Comment by poojapatil36** [comment]
Verified the issue, issue is fixed.
Please find below attachment for reference.
Github tag: #Closed;
![5100_A11y_Visual Studio Code Services_Home_Header_ScreenReader (2)](https://user-images.githubusercontent.com/97668775/218077000-f55617e8-c6e5-40b2-b1f3-0dbd4ef3421a.png)
![5100_A11y_Visual Studio Code Services_Home_Header_ScreenReader](https://user-images.githubusercontent.com/97668775/218077011-18a0f685-06da-4059-a8c1-91bb7f7c1f67.png)
